src/partials/gallery.html

```diff
@@ -0,0 +1,6 @@
+<section class="section-gallery">
+    <div class="container">
+        <img class="section-gallery__image" src="images/gallery.jpg" alt="images with ice-creams"
+        width="364" />
```

Comment:
Zmień proszę nazwę klasy na `gallery__image`

Po co jest ten `width="364"`? Wydaje mi się, że to nie jest potrzebne.